main/busybox-initscripts/busybox-initscripts.post-install

```diff
@@ -9,18 +9,29 @@ if [ -L /etc/runlevels/boot/mdev ] && ! [ -L /etc/runlevels/sysinit/mdev ]; then
 	mv /etc/runlevels/boot/mdev /etc/runlevels/sysinit/ 2>/dev/null
 fi
 
+rename_service() {
+	oldname="$1"
+	newname="$2"
+
+	for link in /run/openrc/*/${oldname} /etc/runlevels/*/${oldname}; do
+		newlink="${link%/*}/${newname}"
+		if [ -L "$link" ] && ! [ -L "$newlink" ]; then
+			target="$(readlink $link)"
+			ln -s "${target%/*}/$newname" "$newlink"
+			rm "$link"
+			continue
+		fi
+		if [ -d "$link" ] && ! [ -d "$newlink" ]; then
+			mv "$link" "$newlink"
+		fi
+	done
+}
+
 # cron was renamed to crond
-for link in /run/openrc/*/cron /etc/runlevels/*/cron; do
-	if [ -L "$link" ] && ! [ -L "${link}d" ]; then
-		target="$(readlink $link)"
-		ln -s "${target}d" "${link}d"
-		rm "$link"
-		continue
-	fi
-	if [ -d "$link" ] && ! [ -d "${link}d" ]; then
-		mv "$link" "${link}d"
-	fi
-done
+rename_service cron crond
+
+# keymaps was renamed to loadkmap
+rename_service keymaps loadkmap
```

Comment:
Do we also want to output some kind of message if the service was renamed? Users still need to adjust their /etc/conf.d/loadkmaps configuration file afterwards and if they made changes to /etc/init.d/keymaps they might need to remove that file manually.

Reply:
Yes, I think it is a good idea.